packages/@react-spectrum/card/src/GridLayout.tsx

```diff
@@ -67,8 +76,21 @@ const DEFAULT_OPTIONS = {
   L: {
     // TODO: for now bumping this higher since the new cards have more stuff in the content area.
     // Will need to ask Spectrum what these values should be. Used to be 52. Do the same for S above
-    itemPadding: 95,
-    minItemSize: new Size(208, 208),
+    itemPadding: {
```

Comment:
still need this TODO?

Reply:
Yah, feel more comfortable keeping it until we get small cards supported as well